clang/test/CIR/CodeGen/vtt.cpp

```diff
@@ -73,19 +84,53 @@ void D::y() {}
 // CIR-SAME:   ]> : !cir.array<!cir.ptr<!u8i> x 4>
 // CIR-SAME: }> : !rec_anon_struct {alignment = 8 : i64}
 
+// CIR-RTTI:      cir.global{{.*}} @_ZTV1D = #cir.vtable<{
+// CIR-RTTI-SAME:   #cir.const_array<[#cir.ptr<40 : i64> : !cir.ptr<!u8i>,
+// CIR-RTTI-SAME:     #cir.ptr<null> : !cir.ptr<!u8i>, 
+// CIR-RTTI-SAME:     #cir.global_view<@_ZTI1D> : !cir.ptr<!u8i>, 
+// CIR-RTTI-SAME:     #cir.global_view<@_ZN1B1wEv> : !cir.ptr<!u8i>,
```

Comment:
```suggestion
// CIR-COMMON-SAME:     #cir.ptr<null> : !cir.ptr<!u8i>, 
// CIR-NO-RTTI-SAME:    #cir.ptr<null> : !cir.ptr<!u8i>, 
// CIR-RTTI-SAME:       #cir.global_view<@_ZTI1D> : !cir.ptr<!u8i>, 
// CIR-COMMON-SAME:     #cir.global_view<@_ZN1B1wEv> : !cir.ptr<!u8i>,
```
This is an example of what I'd like to see for these checks. The first line is the same whether RTTI is enabled or not. The following two lines check the two variations of the output that is expected without and with RTTI enabled. Then the last line here is back to common.

I think that file check will let you mix prefixes like this while still parsing the same line.

Reply:
Done 👍🏻